package.json

```
@@ -37,15 +38,14 @@
    "license": "UNLICENSED",
    "devDependencies": {
       "cross-env": "7.0.3",
-      "husky": "4.2.3",
+      "husky": "^8.0.0",
```

Use the caret, get the stick. 
```suggestion
      "husky": "8.0.1",
```

---

Fixed.